packages/protocol/test/libs/LibTrieProof.test.ts

```diff
@@ -50,19 +50,9 @@ describe("integration:LibTrieProof", function () {
             chainId
         );
 
-        const libBridgeRetry = await (
-            await ethers.getContractFactory("LibBridgeRetry")
-        ).deploy();
-
-        const libBridgeProcess = await (
-            await ethers.getContractFactory("LibBridgeProcess")
-        ).deploy();
-
         const BridgeFactory = await ethers.getContractFactory("Bridge", {
             libraries: {
-                LibBridgeProcess: libBridgeProcess.address,
-                LibBridgeRetry: libBridgeRetry.address,
-                LibTrieProof: testLibTreProof.address,
+                LibTrieProof: testLibTrieProof.address,
```

Comment:
So seems we haven't started testing openzeppelin's MerkleProof library in this PR yet right?

Reply:
No, but @adaki2004 is looking into this for us!

Reply:
@cyberhorsey @davidtaikocha 
I think the OZ's MerkleProof library ([link](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/MerkleProof.sol)) is not suitable here. Maybe there is another library which i could not find, but this one is a verifying library for Merkle Trees, but what we actually need here is Merkle Patricia Tries verifying algo.

Let me know if i misunderstood something or could not find a library. 
(BTW i found [this](https://github.com/pipeos-one/goldengate/blob/master/contracts/contracts/lib/MPT.sol), which can be suitable, do not know if audited or not, but will try to give a shot if it can give us any savings.)

Reply:
Lets continue this convo elsewher as this PR will be merged in soon without the Merkle Trie lib replacement.